deploy.groovy

```diff
@@ -0,0 +1,74 @@
+//Groovy scripts throw lots of WARNING messages, but it is fine
+//https://issues.apache.org/jira/browse/GROOVY-8339
+
+//Edit this file to define the following Strings
+def SET_DOCKER_REG = "ip:port"
+def SET_DOCKER_W = "/path/to/docker/wrapper/"
```

Comment:
Are there immediate plans to define these values in the file? If not, I suggest removing them. Too often we have added flexibility that never gets used

Reply:
This option was in the bash script for people who don't want to set environment variables, they would be easy to remove if we want to make a decision not to support that.

Reply:
These will only get used if a dev doesn't have the values populated as env variables. It might make more sense just to exit the script with an error message if the environment variables aren't populated.

Reply:
I will remove the defaults.